driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -192,8 +191,8 @@ private Cluster(
       List<EndPoint> contactPoints,
       Configuration configuration,
       Collection<Host.StateListener> listeners) {
-    System.out.println("===== Using optimized driver!!! =====");
-    logger.info("===== Using optimized driver!!! =====");
+    System.out.println("===== Using Cloud optimized driver!!! =====");
```

Comment:
I would be nice if this string isn't changed at all.

Since there is code in SCT using this line to tell if it's shared aware driver or not

I don't see any value to knowing if it's cloud related or not (i.e. whom run it with a bundle would know with any log)